cpp/lib/src/ZoteroHarvesterUtil.cc

```diff
@@ -710,6 +718,21 @@ void UploadTracker::registerZederJournal(const unsigned zeder_id, const std::str
 }
 
 
+void UploadTracker::deleteAllDeliveredOnlineFirstEntries(const unsigned zeder_id, const std::string &zeder_instance) {
```

Comment:
It might be useful to keep this function a bit more generic and pass the delivery_state as a parameter.

Reply:
Intentionally done like this. Other states already have logic to be removed/reset, so another approach seems dangerous.